react-backend/database/dbBuild.sql

```diff
@@ -45,7 +45,8 @@ CREATE TABLE activities
   thursday BOOLEAN DEFAULT false,
   friday BOOLEAN DEFAULT false,
   saturday BOOLEAN DEFAULT false,
-  sunday BOOLEAN DEFAULT false
+  sunday BOOLEAN DEFAULT false,
+  date TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
```

Comment:
`date`/`DATE` is a key word in sql, use another column name

Reply:
Coming from our talk I've opened an issue about this - #204 .